util/size_regression.sh

```diff
@@ -49,7 +66,9 @@ function build_executor() {
         revision=$(echo $line | cut -d' ' -f1)
 
         make distclean >/dev/null 2>&1
-        git checkout $revision >/dev/null 2>&1 || { echo "Failed to check out revision ${revision}" >&2 ; exit 1 ; }
+
+        git clean -xfd >/dev/null 2>&1 || true
```

Comment:
The downside here is that it will remove items from global ignore, in my case blow away my direnv config

Reply:
Removed.